optimum_benchmark/backends/utils.py

```diff
@@ -232,3 +272,343 @@ def export_dummy_model(
             raise Exception(
                 f"The post-processing of the ONNX export failed. The export can still be performed by passing the option --no-post-process. Detailed error: {e}"
             )
+
+
+def main_export(
```

Comment:
yes it's simulating the export in `ORTModelForxxx` to skip downloading big LLM weights

Reply:
this was handled in an ugly way before (while trying to reduce code). but since the main_export of optimum changes (with sdxl for example) I decided to keep all of it and only "hikack" the model instantiation by tasks manager.